listenbrainz_spark/__init__.py

```diff
@@ -1,19 +1,13 @@
-from pyspark import SparkContext
-from pyspark.sql import SparkSession, SQLContext
-
+from pyspark.sql import SparkSession
 
 session = None
-context = None
-sql_context = None
 
 def init_spark_session(app_name):
-    global session, context, sql_context
+    global session
     session = SparkSession \
             .builder \
             .appName(app_name) \
             .config("spark.hadoop.dfs.client.use.datanode.hostname", "true") \
             .config("spark.hadoop.dfs.datanode.use.datanode.hostname", "true") \
             .getOrCreate()
-    context = session.sparkContext
-    context.setLogLevel("ERROR")
```

Comment:
Why are we removing this?

I'm pretty sure context and sql_cobtext are used elsewhere